### PR TITLE
Master 1.12

### PR DESCRIPTION
**Pull request by SkyBlade1978**
now basically working, there are bugs!

### CODE REVIEW

File: src/main/java/cyano/mineralogy/Mineralogy.java [hunk 1]
```diff
@@ -111,25 +94,26 @@
 	private static final String fertilizer = "fertilizer";
 	private static final String stone = "stone";
 	private static final String dustCarbon = "dustCarbon";
-	private static final String blockPhosphorous = "blockPhosphorous";
-	private static final String dustPhosphorous = "dustPhosphorous";
+	private static final String blockPhosphorous = "phosphorous_block";
```
Comment (jriwanek) [inline src/main/java/cyano/mineralogy/Mineralogy.java]: HUGE redflag comes up in my mind on this change, these Strings contained oredicts, not block names.
It's looking like you're not passing block names where oredicts should have been passed,

File: src/main/java/cyano/mineralogy/Mineralogy.java [hunk 2]
```diff
@@ -182,10 +160,10 @@ public void preInit(FMLPreInitializationEvent event) {
 		addStoneType(RockType.METAMORPHIC,"amphibolite", 3, 15, 1);
 
 		// add items
-		gypsumPowder = addDust("gypsum_dust", "Gypsum");
-		sulphurPowder = addDust("sulfur_dust", "Sulfur");
-		phosphorousPowder = addDust("phosphorous_dust", "Phosphorous");
-		nitratePowder = addDust("nitrate_dust", "Nitrate");
+		gypsumPowder = addDust(dustGypsum, "Gypsum");
```
Comment (jriwanek) [inline src/main/java/cyano/mineralogy/Mineralogy.java]: Related to previous comment

File: src/main/java/cyano/mineralogy/Mineralogy.java [hunk 3]
```diff
@@ -239,9 +217,9 @@ public void preInit(FMLPreInitializationEvent event) {
 		OreDictionary.registerOre(oreNitrate, nitrateBlock);
 		
 		// TODO: Finish This
-		addBlock("sulfur_block", "Sulfur", 0);
-		addBlock("phosphorous_block", "Phosphorous", 0);
-		addBlock("nitrate_block", "Nitrate", 0);
+		addBlock(blockSulfur, "Sulfur", 0);
```
Comment (jriwanek) [inline src/main/java/cyano/mineralogy/Mineralogy.java]: Related to previous comment

File: src/main/java/cyano/mineralogy/Mineralogy.java (hunk 1 above)
Comment (SkyBlade1978) [reply src/main/java/cyano/mineralogy/Mineralogy.java]: thanks, I'll sort this out later

**Comment by jriwanek** [comment]
Looks alright, but we'll wring out any other bugs in the next PR

**Pull request merged**